src/assets/i18n/uk.json

```diff
@@ -466,5 +505,27 @@
     "NO_MESSAGES": "Повідомлень поки немає"
   },
   "NOTIFICATIONS": "Сповіщення",
-  "APPLICATION_STATUSES": "Статуси заявок"
+  "APPLICATION_STATUSES": "Статуси заявок",
+  "HISTORY_LOG": {
+    "WAS": "Було",
+    "BECAME": "Стало",
+    "SHOW" : "Показати",
+    "RESET_FILTERS": "Скинути фільтри",
+    "PERIOD": "Період",
+    "REMOVE_ADMIN_OPTION": "Видалення адміністратора з гуртка",
+    "ADD_ADMIN_OPTION": "Створення адміністратора"
+  },
+  "STATISTICS": {
+    "DATE_RANGE": "Часовий проміжок",
+    "FORMAT": "Формат",
+    "GENERATE_REPORT": "Згенерувати звіти",
+    "NO_DATA": "У вас ще немає згенерованих звітів.",
+    "NO_DATA_SUBHEADER":"Задайте критерії і натисніть кнопку “Згенерувати звіти”, щоб побачити їх тут",
+    "TITLE": "Назва",
+    "PERIOD": "Період",
+    "CREATE_DATE": "Дата створення",
+    "DOWNLOAD": "Завантажити",
+    "WORKSHOPS_DAILY": "За останні 24 години",
+    "WORKSHOPS_YEAR": "За рік"
```

Comment:
You forgot to add the translation WORKSHOPS_DAILY and WORKSHOPS_YEAR to the English version, please add